docs/pages/management/operations/backup-restore.mdx

```diff
@@ -202,6 +210,19 @@ $ tctl --config fresh.yaml get all
 # <your state here>
 ```
 
+<Notice>
```

Comment:
Same comments as the other use of this `Notice`.

Reply:
@stevenGravy This comment still needs to be addressed. We should edit this `Notice` to match the other `Notice`.